### ISSUE TITLE
ReferenceError: utils_1 is not defined

### ISSUE THREAD

**Issue by Brinsky** [ranmocy/gmail-automata#61]
The [current script](https://script.google.com/home/projects/1yFWTUxIMOM1oXZTkM4dGLFypM83yBFWxBZBHrEcpRz688AVElAcuQu-T/edit) attached to the [sheet](https://docs.google.com/spreadsheets/d/1pkx69yw7_gjujuqTPuWhpMiW481RzCeLBizkq0HczcI/edit#gid=1407779459) linked in the README seems to have some errors. Symbols that come from the `utils.gs` file are referenced with `(0, utils_1.assert)` and similar instead of directly by name. This results in `ReferenceError: utils_1 is not defined` when running the script.

**Comment by mlsad3** [comment]
This appears to be a duplicate of #59  (which has a PR ready to go)

**Comment by Deybat** [comment]
Hey @Brinsky @mlsad3 
Is there a solution for "ReferenceError: utils_1 is not definedDetails"

**Comment by mlsad3** [comment]
Yes, just add PR #59 from @aaronj1335 . It resolves this issue.

**Comment by Deybat** [comment]
@mlsad3 is there a way to update the old script code with fix from @aaronj1335 directly in script.google.com and make it work with spreadsheets attache to it.

**Comment by mlsad3** [comment]
Yes.... you can, but it requires taking that updated sheet, opening the Extensions->Apps Script page, and copying the content of each of the scripts into your own (and making any new scripts if they don't already exist).

I made some potential updates for the spreadsheet to help describe how to update. See attached images below, in particular, the "Updating" section:

![gmailautomata1](https://user-images.githubusercontent.com/15711477/184277837-3124d515-17e5-4599-a4ee-456d7b1004aa.png)
![gmailautomata2](https://user-images.githubusercontent.com/15711477/184277841-dea09dbe-913f-4b0d-aad1-e4f68efb3096.png)

**Comment by Deybat** [comment]
Ohh!! @mlsad3 **Thank you** for this awesome instructions

i did try to update script code manually after using an online compiler to turn ts to js (didn't want to setup local Dev Environment)
after saving the project i lost 'Gmail Automata' button from the main menu in the spreadsheet(i tried adding trigger as suggested in the README but didn't work)

i'll try to setup clasp and test again.

**Comment by mlsad3** [comment]
If 'Gmail Automata' disappears, it's probably because your clasp setup worked and there is a script error. When it initializes the first time, it runs part of the script.

Click on 'Extensions' -> 'Apps Script', then inside the Apps Script, select the left menu "Executions" to analyze the log files (if any) of the extension.

![image](https://user-images.githubusercontent.com/15711477/184380357-947019b9-31cb-4a3a-9186-cf3a9fc0d7e0.png)

Look for any mention of failure. Sometimes if it's too recent (a few seconds), you can click the refresh button to see if any logs were created.

**Comment by Deybat** [comment]
Yes, you're right
but am lost here not sure why it give that error
![Screenshot from 2022-08-15 13-16-16](https://user-images.githubusercontent.com/110777847/184633917-d6e885cc-1292-4a42-9bee-8d66d56b5e76.png)

**Comment by mlsad3** [comment]
Hmm,

Looks like your online flow does not utilize [the .claspignore](https://github.com/ranmocy/gmail-automata/blob/main/.claspignore) file, and it deployed the `Condition.test.ts` file. That should not have happened, as they require Jest, and none of those libraries are ever deployed.

**Comment by Deybat** [comment]
i did remove Condition.test.ts and the Gmail Automata button appear now but when trying to "Process Now" it fails 
![Screenshot from 2022-08-15 19-55-24](https://user-images.githubusercontent.com/110777847/184698588-c9980641-e149-4e75-b6b3-0db5f546be79.png)

**Comment by mlsad3** [comment]
PR #59 recently (today) was merged, and should resolve that

**Comment by Deybat** [comment]
Yes it did work now thanks again @mlsad3 for walking me through this. You're awesome.
